release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -475,6 +475,61 @@ For more information, see xref:../registry/configuring_registry_storage/configur
 [id="ocp-4-17-storage_{context}"]
 === Storage
 
+[id="ocp-4-17-storage-efs-vol-metrics"]
+==== AWS EFS storage CSI usage metrics is generally available
+Amazon Web Services (AWS) Elastic File Service (EFS) metrics allow you to monitor how much space is used by EFS volumes. This feature is generally available.
+
+[IMPORTANT]
+====
+Turning on metrics can lead to performance degradation because the CSI driver walks through the whole volume. Therefore, this option is disabled by default. Administrators must explicitly enable this feature.
+====
+
+// For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-aws-efs.html#efs-metrics-procedure_persistent-storage-csi-aws-efs[Amazon Elastic File Storage CSI usage metrics].
+
+[id="ocp-4-17-storage-control-vol-mode-conversion"]
+==== Preventing unauthorized volume mode conversion is generally available
+Previously, there was no validation of whether the mode of an original volume (filesystem or raw block), whose snapshot was taken, matches the mode of a newly created volume. This presented a security gap that could allow malicious users to potentially exploit an as-yet-unknown vulnerability in the host operating system.
+
+Nevertheless, some users have a legitimate need to perform such conversions. This feature allows cluster administrators to provide these rights (ability to perform update or patch operations on VolumeSnapshotContents) only to trusted users or applications, such as backup vendors.
+
+To convert a volume mode, an authorized user needs to change `snapshot.storage.kubernetes.io/allow-volume-mode-change: "true"` for VolumeSnapshotContent of the snapshot source.
+
+This feature is supported as generally available.
+
+[id="ocp-4-17-storage-auto-del-resources-gcp-filestore"]
+==== Automatic deletion of resources for GCP Filestore is generally available
+In earlier versions of {product-title}, when destroying a cluster, Google Compute Platform (GCP) Filestore Storage did not delete all of the cloud resources belonging to that cluster. This required manually deleting all of the persistent volume claims (PVCs) that used the Filestore storage class before destroying the cluster.
+
+With {product-title} 4.17, when destroying a cluster the {product-title} the installer should generally delete all of the cloud resources that belong to that cluster, and therefore manual deletion of PVCs should not be required. However, due to the special nature of the Google Compute Platform (GCP) Filestore resources, the automated cleanup process might not remove all of resources in some cases. This feature is supported as generally available.
```

Comment:
Is "the {product-title} the installer" a typo? Do you mean "{product-title} installer"?